From bed1568548d7a47785d8fe24ec7a9ce5799040ff Mon Sep 17 00:00:00 2001
From: Guilherme Amadio <amadio@cern.ch>
Date: Wed, 27 Jun 2018 15:08:07 +0200
Subject: [PATCH] Do not exclude etc/http directory from installation

This was an unintended side-effect of a previous commit:
9b4d0d8dccbd48d21b0f7c79b6eaf94428f691b1.
---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c760d0fdb..a3a0f59925 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -426,8 +426,7 @@ if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_INSTALL_PREFIX)
                          PATTERN "rootd.xinetd" EXCLUDE
                          PATTERN "proofd.xinetd" EXCLUDE
                          PATTERN "root.mimes" EXCLUDE
-                         PATTERN "cmake" EXCLUDE
-                         PATTERN "http" EXCLUDE )
+                         PATTERN "cmake" EXCLUDE )
   install(DIRECTORY fonts/  DESTINATION ${CMAKE_INSTALL_FONTDIR})
   install(DIRECTORY icons/  DESTINATION ${CMAKE_INSTALL_ICONDIR})
   install(DIRECTORY macros/ DESTINATION ${CMAKE_INSTALL_MACRODIR})
-- 
2.18.0

